src/components/Loginn/TokenValid.js

```diff
@@ -0,0 +1,33 @@
+import { useNavigate } from 'react-router-dom';
+import {  useEffect } from 'react';
+
+
+
+function TokenValid() {
+
+  const navigate = useNavigate();
+  useEffect(() => {
+    const token = localStorage.getItem('token');
+    if (token && isTokenValid(token)) {
+        console.log('Користувач авторизований');
+        navigate('/');
+    }
+}, [navigate]);
+const isTokenValid = (token) => {
```

Comment:
You don't need to validate token, you should just save token in local storage and then attach it to all requests that UI sends to API